cli/azd/extensions/azure.coding-agent/go.mod

```diff
@@ -5,41 +5,45 @@ go 1.25
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.19.1
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.12.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/msi/armmsi v1.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/msi/armmsi v1.3.0
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v1.2.0
-	github.com/azure/azure-dev v0.0.0-20250926202741-c52bbe902f79
+	github.com/azure/azure-dev v0.0.0-20250930162725-a83f227fe10c
```

Comment:
This will get interesting 😁🤓

Reply:
Yeah, perhaps it would be better to start tagging, but still keep it pre-release (ie: v0.0.1) or similar.